es/es-repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureRepository.java

```diff
@@ -118,6 +194,20 @@ public AzureRepository(RepositoryMetaData metadata, Environment environment,
         }
     }
 
+    static boolean checkRepositoryCredentials(Settings repositorySettings) {
```

Comment:
Since both settings are returned under `mandatorySettings`, isn't this unnecessary?